airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/task/implementor/ProcessRecordsTask.kt

```diff
@@ -150,7 +150,7 @@ data class FileAggregateMessage(
 @Secondary
 class DefaultProcessRecordsTaskFactory(
     private val config: DestinationConfiguration,
-    private val deserializer: ProtocolMessageDeserializer,
+    @Named("processRecordsDeserializer") private val deserializer: ProtocolMessageDeserializer,
```

Comment:
nonblocking: is it possible to do `ProtocolMessageDeserializer<DestinationRecordAirbyteValue>` to avoid the named bean? (I'm guessing this is nontrivial, just want to doublecheck)

Reply:
I changed it so the deserializer deserializes to `AirbyteRecord`

then you can call `asRecordSerialized` or `asRecordMarshaledToAirbyteValue`. Cleaner and way more type safe.